packages/v4/patternfly-docs/content/training/html-css/fundamentals.md

```diff
@@ -156,7 +156,7 @@ Adding the `pf-m-gutter` modifier class to the grid layout ensures there is equa
 
 Add the class `pf-m-gutter` after `pf-v5-l-grid` to the outermost wrapper, inside the quotation marks.
 It should look like: `<div class="pf-v5-l-grid pf-m-gutter">`
-**Note:** Learn how modifier classes work with layout classes by looking at PatternFly's <a href="/layouts/grid/html#usage" target="_blank">grid documentation</a>.
+**Note:** Learn how modifier classes work with layout classes by looking at PatternFly's <a href="/layouts/grid/html" target="_blank">grid documentation</a>.
```

Comment:
hm.... i think we want to keep the `#usage` in this url. It's not working at the moment because this URL was missing the `/v4/` in the href - but Katie is actively working to remove that from all URLS, so these links will fix themselves very soon I think. 

This change can be reverted.